api/tests/routes/backoffice_v3/finance_test.py

```diff
@@ -46,7 +49,7 @@ def test_list_incidents_without_filter(self, authenticated_client):
 
         rows = html_parser.extract_table_rows(response.data)
         assert len(rows) == len(incidents)
-        assert rows[0]["ID"] == str(incidents[0].id)
+        assert rows[0]["ID"] == str(incidents[-1].id)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Je n'ai pas compris pourquoi l'index est à `-1`.

Reply:
C'est juste pour prendre le dernier élément de la liste sachant que j'ai inversé l'ordre d'affichage des incidents, il fallait que je mette à jour le test en conséquence

Reply:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Est-ce qu'il ne vaudrait mieux pas indiquer qu'il y a un ordre inverser entre `rows` et `incidents` ? Genre `reversed_rows` ou `rows_desc` ?

Reply:
Hum je suis en train d'y réfléchir mais perso je laisserai `rows` comme ca. On veut vraiment checker la premier ligne. En revanche je peux encapsuler `incidents[-1]` dans une variable `last_created_incident` pour plus de clarté